wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -234,11 +234,11 @@ public protocol UserSession: AnyObject {
 
     var updateMLSGroupVerificationStatus: UpdateMLSGroupVerificationStatusUseCaseProtocol { get }
 
-    var lastE2EIUpdateDate: LastE2EIdentityUpdateDateProtocol? { get }
+    var lastE2EIUpdateDate: LastE2EIdentityUpdateDateRepositoryInterface? { get }
```

Comment:
without the repository suffix, I would tend to read it's a date
```suggestion
    var lastE2EIUpdateDateRepository: LastE2EIdentityUpdateDateRepositoryInterface? { get }
```

Reply:
done